hazelcast/src/hazelcast/client/network.cpp

```diff
@@ -1407,7 +1406,7 @@ namespace hazelcast {
                     std::vector<CipherInfo> supportedCiphers;
                     for (int i = 0; i < sk_SSL_CIPHER_num(ciphers); ++i) {
                         struct SSLSocket::CipherInfo info;
-                        SSL_CIPHER *cipher = const_cast<SSL_CIPHER *>(sk_SSL_CIPHER_value(ciphers, i));
+                        const SSL_CIPHER *cipher = sk_SSL_CIPHER_value(ciphers, i);
```

Comment:
@ihsandemir Can you double-check here? I am not confident with this as I didn't understand why this cast was used in the first place (maybe it used to be necessary, but now it's not.). I just removed the cast and declared the pointer as pointer-to-const-variable and it compiled.

Reply:
I looked at the declarations of all the functions that we pass the pointer `cipher` here (SSL_CIPHER_get_name, SSL_CIPHER_get_bits, ..). They all take the cipher as `const SSL_CIPHER *c`, so I am sure it's fine to for the cipher to be const here. Merging it this way.